ads/inabox/inabox-messaging-host.js

```diff
@@ -197,14 +191,13 @@ export class InaboxMessagingHost {
    *
    * @param {!Object} request
    * @param {!Window} source
-   * @param {string} origin
    * @param {?JsonObject} data
    */
-  sendPosition_(request, source, origin, data) {
+  sendPosition_(request, source, data) {
```

Comment:
shall we still pass down `origin` and do `*` only when it is `null`?

cc @molnarg to chime in from security's perspective

Reply:
Based on the PR description, it's public data so it looks fine to leave it like this. Please add a comment that explains it.

Reply:
added comment

Reply:
@molnarg still want to throw my 2c. 

I don't think user's scroll position data are really public data. We do have some mechanism in place to make sure only iframes created by the ads tag on the page can request the data. we also use sentinel token to make sure the integrity in those iframes.

My own principal for security-wise thing is the tighter constraint the better, if it does not come with big trade-off.

Reply:
@lannka I can't tell from your comment whether you're ok with this PR?

Reply:
I'm not convinced that we should remove this extra check as the assumption that "data is public" is wrong.

Reply:
Chat with @jeffkaufman offline and I am convinced there is no extra security benefit here to set the `origin`.